Doc/library/sqlite3.rst

```diff
@@ -52,51 +52,54 @@ This document includes four main sections:
 Tutorial
 --------
 
-To use the module, start by creating a :class:`Connection` object that
-represents the database.  Here the data will be stored in the
-:file:`example.db` file::
+In this tutorial you will learn the basics of the :mod:`!sqlite3` API
+by creating an on-disk database :file:`example.db`,
+and executing SQL queries against it.
+A fundamental understanding of database concepts,
+like transactions and cursors, is assumed.
+
+Start by using the :func:`sqlite3.connect` function to
+open a database :class:`Connection` to :file:`example.db`;
+SQLite will implicitly create the database file :file:`example`
+in the current working directory, if it does not exist.
+The returned :class:`!Connection` object represents our
+connection to the on-disk SQLite database::
 
    import sqlite3
    con = sqlite3.connect('example.db')
 
-The special path name ``:memory:`` can be provided to create a temporary
-database in RAM.
-
-Once a :class:`Connection` has been established, create a :class:`Cursor` object
-and call its :meth:`~Cursor.execute` method to perform SQL commands::
+Now, create a :class:`Cursor` object using :meth:`~Connection.cursor`.
+Call its :meth:`~Cursor.execute` method to perform SQL queries::
```

Comment:
> Alternatively, this could be split into two different code blocks, with corresponding descriptions, especially if more of an intro is added there.

That may be needed; for now, I think it is good as it is.